tutor/routes/course.py

```diff
@@ -5,3 +5,8 @@
 @app.route('/course/<id>')
 def get_course_info_route(id):
     return course.showCoursePage(id)
+
```

Comment:
shouldn't resources-related routes go in a resources.py file rather then a courses.py file?